2022-06-29 11:52:25,118 INFO  ==============================================================
2022-06-29 11:52:25,119 INFO  Id:    log_request_error_messages
2022-06-29 11:52:25,120 INFO  Dir:   log_request_end_to_end/
2022-06-29 11:52:25,121 INFO  Title: Validate end to end behaviour for the log request operation check for error messages
2022-06-29 11:52:25,122 INFO  ==============================================================
2022-06-29 11:52:26,637 INFO  EnvironmentC8y Setup
2022-06-29 11:52:27,403 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-06-29 11:52:41,884 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 14 secs
2022-06-29 11:52:42,508 INFO  Executed sudo<tedge_connect_c8y_test>, exit status 0
2022-06-29 11:52:43,122 INFO  Executed systemctl<serv_mosq2>, exit status 0
2022-06-29 11:52:43,693 INFO  Executed systemctl<serv_mapper3>, exit status 0
2022-06-29 11:52:44,209 INFO  Executed sudo<rm_logs>, exit status 0
2022-06-29 11:52:44,803 INFO  Executed sudo<move_logs>, exit status 0
2022-06-29 11:52:45,434 INFO  Executed sudo<log_file_daemon>, exit status 0
2022-06-29 11:52:45,960 WARN  TypeError occurred while running test - 'NoneType' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/run.py", line 47, in execute
    log_file_request_payload, self.project.deviceid
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/environments/cumulocity.py", line 101, in trigger_log_request
    "deviceId": self.device_fragment["id"],
TypeError: 'NoneType' object is not subscriptable
2022-06-29 11:52:46,562 WARN  TypeError: 'NoneType' object is not subscriptable ... blocked 
2022-06-29 11:52:46,563 INFO  
2022-06-29 11:52:46,563 INFO  cleanup:
2022-06-29 11:52:47,075 INFO  Executed sudo<rm_logs>, exit status 0
2022-06-29 11:52:47,641 INFO  Executed sudo<rm_tmp_logs>, exit status 0
2022-06-29 11:52:48,307 INFO  Executed sudo<send_failed>, exit status 0
2022-06-29 11:52:48,920 INFO  Executed sudo<log_file_daemon>, exit status 0
2022-06-29 11:52:52,832 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-06-29 11:52:53,398 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-06-29 11:52:53,402 INFO  
2022-06-29 11:52:53,403 INFO  Test duration: 28.29 secs
2022-06-29 11:52:53,404 INFO  Test final outcome:  BLOCKED
2022-06-29 11:52:53,405 INFO  Test outcome reason: TypeError: 'NoneType' object is not subscriptable
2022-06-29 11:52:53,406 INFO  
